mbed/mbed.py

```diff
@@ -1447,6 +1440,22 @@ def ignore_build_dir(self):
             except IOError:
                 error("Unable to write build ignore file in \"%s\"" % os.path.join(build_path, '.mbedignore'), 1)
 
+    def detect_target(self, info=None):
+        targets = self.get_detected_targets()
+        if targets == False:
+            error("The target detection requires that the 'mbed-ls' python module is installed.\nYou can install mbed-ls by running 'pip install mbed-ls'.")
+        elif len(targets) > 1:
+            error("Multiple targets were detected.\nOnly 1 target board should be connected to your system.")
+        elif len(targets) == 0:
+            error("No targets were detected.\nPlease make sure a target board is connected to this system.")
```

Comment:
On linux, it's not enough to connect the target board to the system. You must also mount the fake file system of the board somewhere. Should we mention this in the error message?